### DISCUSSION TITLE
Direct field interaction on interfaces

### DISCUSSION THREAD

**Discussion by Guiqft** [directus/directus#7058] [Archive]
How can I get another field value to use in my interfaces?
as example: 
I have one field `species` to select a specie on the list and, on my interface, I wanna use the selected specie to filter another list. 
This behavior should be on the same item page, with the first field direct changing and affecting which values my interface will show.

**Comment by rijkvanzanten** [comment]
This is very similar to https://github.com/directus/directus/discussions/7071#discussion-3489023 🙂 

My answer from over there:

> > Is there a native way to make this filtering?
>
> Yes! I just released actually (rc.86+). Under field setup, there's a new tab called "Conditions". In these conditions, you can configure changes to the field based on the values of the other fields in the form, so you could update the dropdown options of "Municipalities" based on the selected "Countries" value

I'll be adding some more in-depth docs soon: https://github.com/directus/directus/issues/7073
